### PR TITLE
[a11y] - Content Types Preference description can't be read by JAWS

### LINKED ISSUE #340: [a11y] - Content Types Preference description can't be read by JAWS
Steps to reproduce on Windows:
1. Open JAWS
2. Launch Eclipse
3. Click Window > Preferences > General > Content Types
Eclipse_Preference_ContentType_JAWS

===== EXPECTED RESULT =====
The content should be read out for user.

### PR DESCRIPTION
**Pull request by niraj-modi**
- Fixes #340 

Signed-off-by: Niraj Modi <niraj.modi@in.ibm.com>

### CODE REVIEW

File: bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Link.java [hunk 1]
```diff
@@ -105,8 +106,19 @@ public class Link extends Control {
...
+	super(parent, style);
+	/*
+	 * Accessible tool like JAWS by default only reads the hypertext link text and
+	 * not the non-link text. To make JAWS read the full text we need to tweak the
+	 * default behavior and explicitly return the full link text as below.
+	 */
+	this.getAccessible().addAccessibleListener(new AccessibleAdapter() {
+		@Override
+		public void getName(AccessibleEvent e) {
+			e.result = Link.this.getText();
```
Comment (lshanmug) [inline bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Link.java:119]: We can use text here directly instead of Link.this.getText().

**Review by lshanmug — APPROVED** [review]
Looks good to me. Changes are minimal and it fixes an accessibility issue.

File: bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Link.java (hunk 1 above)
Comment (niraj-modi) [reply bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Link.java:119]: Thanks Lakshmi, will do the refactoring after RC1 is promoted

Comment (niraj-modi) [reply bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Link.java:119]: #353 takes care of this.

**Pull request merged**